packages/forms/backend/sirius-components-forms-graphql/pom.xml

```diff
@@ -74,6 +74,11 @@
 			<artifactId>spring-boot-starter-test</artifactId>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
```

Comment:
Please move the dependency before the `sirius-components-test` one. Sorry I didn't see that before.